.gitattributes

```diff
@@ -9,3 +9,5 @@
 /.travis.yml        export-ignore
 /.scrutinizer.yml   export-ignore
 /appveyor.yml       export-ignore
+
+/tests/_data/semver.crlf   text eol=crlf
```

Comment:
-1 on this. Rather than require just one file has Windows line endings, it would be better to assume that all of the text fixtures are consistently stored with Unix line endings. Then, in the tests, read in the fixture, convert the line endings, and write it to a tmp file to use in the test.

Reply:
OK, I'll fix it.